includes/bash_profile

```diff
@@ -27,9 +27,9 @@ if [ ! -f "/usr/bin/raspi-config" ]; then
   alias raspi-config="echo 'raspi-config is not part of openHABian, please use openhabian-config instead.'"
 fi
 
-if [ -f /opt/FireMotD/FireMotD ]; then
+if [ -f /usr/local/bin/FireMotD ]; then
   echo
-  bash /opt/FireMotD/FireMotD -HV --theme gray
+  FireMotD --theme Gray
```

Comment:
Mixed absolute and relative path.

The conditional call is not included in the line above. I think we can remove the condition here.

Reply:
Well we need to settle in on relative vs absolute paths globally in this project. 
I don't know what is best but using relative gives us some more headroom for old installation maybe using different paths compared to new installations.

And you we should be able to remove the if case as all installation using the bash_profile will have FireModD installed.